pages/_app.tsx

```diff
@@ -12,6 +23,29 @@ function MyApp({ Component, pageProps }: AppProps) {
       </Head>
       <Component {...pageProps} />
       <Analytics />
+      {showGitInfo ? (
+        <>
+          <Card
+            className="w-fit h-fit bg-light fixed bottom-2 right-2 rounded-full"
+            elevation={darkModeElevation}
+          >
+            <Tooltip title="Open GitHub commit in a new tab">
```

Comment:
Could this be "Open GitHub commit for this instance" or something a bit more precise?

Reply:
Changed